compiler/src/main/scala/dx/compiler/Compiler.scala

```diff
@@ -384,7 +386,15 @@ case class Compiler(extras: Option[Extras],
     ): (DxWorkflow, JsValue) = {
       logger2.trace(s"Compiling workflow ${workflow.name}")
       val workflowCompiler =
-        WorkflowCompiler(extras, parameterLinkSerializer, useManifests, dxApi, logger2)
+        WorkflowCompiler(
+            extras = extras,
+            parameterLinkSerializer = parameterLinkSerializer,
+            useManifests = useManifests,
+            dxApi = dxApi,
+            logger = logger2,
+            project = project,
```

Comment:
It looks like project and folder are not used in WorkflowCompiler, so they do not need to be passed to the constructor.

Reply:
Removed for now. Might add them back to WorkflowCompiler in APPS-623 (depending on cloning discussion), agree they don't need to be in superclass.

Reply:
I still see them?

Reply:
Fixed for WorkflowCompiler.